### ISSUE TITLE
Which interpolation method should be used to resize HSI images?

### ISSUE THREAD

**Issue by bw4sz** [weecology/DeepTreeAttention#57]
HSI is normalized here

https://github.com/weecology/DeepTreeAttention/blob/5627265b9424955fc71a8be7949114427421be4c/src/data.py#L196

and resized here

https://github.com/weecology/DeepTreeAttention/blob/5627265b9424955fc71a8be7949114427421be4c/src/data.py#L214

**Comment by bw4sz** [comment]
https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.scale.html is the scaling method. @MarconiS

**Comment by MarconiS** [comment]
late to the party here. Is it still an open issue to discuss or you decided not to scale features, given that it was worsening the accuracy?
